learning-bitmark/working-with-bitmarks/transferring-bitmarks.md

```diff
@@ -0,0 +1,358 @@
+
+* Submit a transfer request
+
+    ```shell
+    $ bitmark-cli -n <network> -i <sender identity>\
+      transfer -r <receiver> \
+      -t <txid> -u
+    ```
+    > The `transfer` command is to submit a transfer transaction to the network. The options:
+    >
+    >* `receiver` - The identity of the recipient
+    >
+    > `txid` - The id of the last transaction of the bitmark which is transferring.
+    >
+    >**NOTE: ** A transfer transaction on the Bitmark blockchain requires a transaction fee of 0.0002 BTC or 0.002 LTC. Therefore, after the `transfer` command, the user needs to execute the payment for the transaction to be confirmed on the blockchain. The payment information is included in the output of the `transfer` command.
+
+    *Example:*
+
+    ```shell
+    $ bitmark-cli -n <network> -i first\
+      transfer -r second \
+      -t 7e3337ae7596864e6dfd918c07780480ef80cea96fa039ed17d35c3849fcb3ca -u
+    ```
+    ```json
+    {
+      "transferId": "4656604222152d08606d42a40c8590c2100c177cafe374ea90bae30c5bd371e0",
+      "bitmarkId": "7e3337ae7596864e6dfd918c07780480ef80cea96fa039ed17d35c3849fcb3ca",
+      "payId": "d819cff364b9211093fe09c2b462bdd05154472a72fac91a882a8f1129674dc92ac5d2724c8d26b16d414de8fbc5c62e",
+      "payments": {
+        "BTC": [
+          {
+            "currency": "BTC",
+            "address": "mr8DEygRvQwKfP4sVZuHVozqvzW89e193j",
+            "amount": "20000"
+          }
+        ],
+        "LTC": [
+          {
+            "currency": "LTC",
+            "address": "mzkCaHJmu1gdnsL9jxW2bwqtw2MCCy66Ds",
+            "amount": "200000"
+          }
+        ]
+      }
+    }
+    ```
+
+<br>
+
+* Paying for the bitmark transfer using the Bitmark-Wallet
+
+    ```shell
+    #Run the bitcoind
+    $ bitcoind -datadir=<bitcoind config dir>
+
+    # OR run the litecoind
+    $ litecoind -datadir=<litcoind config dir>
+
+    #Pay by BTC
+    $ bitmark-wallet --conf <Bitmark-Wallet config file> btc --<btc network> \
+      sendmany --hex-data '<payId>' '<btc address>,<btc amount in satoshi>'
+
+    #OR Pay by LTC
+    $ bitmark-wallet --conf <Bitmark-Wallet config file> ltc --<ltc network> \
+      sendmany --hex-data '<payId>' '<ltc address>,<ltc amount in photon>'
+
+    ```
+
+    >To execute a bitcoin or litecoin transaction on the local environment, the bitcoind or litecoind need to be running.
+    >
+    > - `payId` - The payment id for the share creation transaction, it is printed in the output of share creation command
+
+    <br>
+
+    > **NOTE:** The payment is able to be performed using another tool which allows users to add the exact hex data to an OP_RETURN as part of the coin transaction
+
+    *Example of paying by ltc:*
+
+    ```shell
+    # run litecoind
+    $ bitcoind -datadir=${HOME}/.config/litecoin/
+
+    # Perform payment
+    $ bitmark-wallet --conf ${XDG_CONFIG_HOME}/bitmark-wallet/test/test-bitmark-wallet.conf \
+      ltc --testnet \
```

Comment:
The ${XDG_CONFIG_HOME} is a new variable that we haven't seen before.

Reply:
Updated.